atd-vzv/src/views/summary/CrashesByTimeOfDay.js

```diff
@@ -223,6 +226,18 @@ const CrashesByTimeOfDay = () => {
                 }
               />
             }
+            xAxis={
+              <LinearXAxis
+                type="category"
+                axisLine={null}
+                tickSeries={
+                  <LinearXAxisTickSeries
+                    line={null}
+                    label={<LinearXAxisTickLabel padding={navigator.userAgent.includes("Firefox") ? 15 : 5} />}
```

Comment:
:( = what was the core issue here?

Reply:
Thanks for checking in! The X axis labels were being partially covered up by the chart when viewed in Firefox (see the associated issue for a screenshot), so we added some additional padding for Firefox users. I noticed there is an [open bug report](https://github.com/jask-oss/reaviz/issues/121) for this problem and this seems to fix it.